docs/topics/managedcluster.md

```diff
@@ -0,0 +1,147 @@
+machine pool, since AKS requires at least one system pool at creation
+time.
+
+## Deploy with clusterctl
+
+A clusterctl flavor exists to deploy an AKS cluster with CAPZ. This
+flavor requires the following environment variables to be set before
+executing clusterctl.
+
+```bash
+# Kubernetes values
+export CLUSTER_NAME="foobar"
+export WORKER_MACHINE_COUNT=2
+export KUBERNETES_VERSION="1.17.4" # NOTE: no preceding "v", unlike normal for CAPI.
+
+# Azure values
+export AZURE_LOCATION="southcentralus"
+export AZURE_RESOURCE_GROUP="${CLUSTER_NAME}"
+export AZURE_SUBSCRIPTION_ID="$(cat ~/sp.json | jq -r .subscriptionId | tr -d '\n')"
+export AZURE_NODE_MACHINE_TYPE="Standard_D8s_v3"
+```
+
+Execute clusterctl to template the resources, then apply to a management cluster:
+
+```bash
+# replace v0.4.4 with the appropriate version you're using
+clusterctl config cluster my-cluster --kubernetes-version v1.17.4 --flavor aks -i azure:v0.4.4 > cluster.yaml
+
+# assumes an existing management cluster
+kubectl apply -f cluster.yaml
+
+# check status of created resources
+kubectl get cluster-api -o wide
+```
+
+## Specification
+
+We'll walk through an example to view available options.
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  name: my-cluster
+spec:
+  clusterNetwork:
+    services:
+      cidrBlocks:
+      - 192.168.0.0/16
+  controlPlaneRef:
+    apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AzureManagedControlPlane
+    name: my-cluster-control-plane
+  infrastructureRef:
+    apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+    kind: AzureManagedCluster
+    name: my-cluster
+---
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureManagedControlPlane
+metadata:
+  name: my-cluster-control-plane
+spec:
+  defaultPoolRef:
+    name: agentpool0
+  location: southcentralus
+  resourceGroup: foo-bar
+  sshPublicKey: ${BASE64_SSH_PUBLIC_KEY}
+  subscriptionID: fae7cc14-bfba-4471-9435-f945b42a16dd # fake uuid
+  version: v1.17.4
+  networkPolicy: azure # or calico
+  networkPlugin: azure # or kubenet
+---
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureManagedCluster
+metadata:
+  name: my-cluster
+spec:
+  subscriptionID: fae7cc14-bfba-4471-9435-f945b42a16dd # fake uuid
+---
+apiVersion: exp.cluster.x-k8s.io/v1alpha3
+kind: MachinePool
+metadata:
+  name: agentpool0
+spec:
+  clusterName: my-cluster
+  replicas: 2
+  template:
+    spec:
+      clusterName: my-cluster
+      infrastructureRef:
+        apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+        kind: AzureManagedMachinePool
+        name: agentpool0
+        namespace: default
+      version: v1.17.4
+---
+apiVersion: exp.infrastructure.cluster.x-k8s.io/v1alpha3
+kind: AzureManagedMachinePool
+metadata:
+  name: agentpool0
+spec:
+  osDiskSizeGB: 512
+  sku: Standard_D8s_v3
+```
+
+The main features for configuration today are networkPolicy and
```

Comment:
let's link to https://docs.microsoft.com/en-us/azure/aks/concepts-network#network-policies and https://docs.microsoft.com/en-us/azure/aks/concepts-network#azure-virtual-networks for further info